DigitalLearningSolutions.Data/Services/TutorialContentService.cs

```diff
@@ -59,6 +59,8 @@ FROM Tutorials
                      AND CustomisationTutorials.CustomisationID = @customisationId
                      AND Tutorials.SectionId = @sectionId
                      AND Tutorials.TutorialID = @tutorialId
+                     AND Customisations.Active = 1
+                     AND CustomisationTutorials.Status = 1

```

Comment:
Might be worth tests to check we won't fetch tutorials in a custimisation which isn't active and that we won't fetch tutorials with CustomisationTutorials.Status != 1. But happy for that to be done with your other SQL query changes 🙂

Reply:
Added those tests in b129eef 🙂